azkaban-web-server/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -591,6 +593,24 @@ private void ajaxScheduleCronFlow(final HttpServletRequest req,
       return;
     }
 
+    final boolean hasFlowTrigger;
+    try {
```

Comment:
Don't understand what does this part of code do?

Reply:
this part is to disallow manual scheduling flows with flow trigger